File: DialedInCoffeeTests/Fixtures/SampleData.swift
import Foundation
@testable import DialedInCoffee

enum SampleData {
    static let baselineFlatWhite = MachineSettings(
        grinder: 4, strength: 7, volumeML: 35, milkSeconds: 18, temperature: .normal
    )

    static let bean = BeanSnapshot(
        id: "ona-maple",
        name: "Maple 🥛",
        roaster: "ONA Coffee",
        roastLevel: .medium,
        milkFriendly: true,
        flavourNotes: ["caramel", "biscuit", "nut", "spice"],
        recommendedSettings: baselineFlatWhite
    )
}

File: android/core/src/test/kotlin/com/cawfee/domain/SampleData.kt
package com.cawfee.domain

import com.cawfee.domain.model.BeanSnapshot
import com.cawfee.domain.model.MachineSettings
import com.cawfee.domain.model.RoastLevel

/** Test fixtures mirroring DialedInCoffeeTests/Fixtures/SampleData.swift. */
object SampleData {
    val baselineFlatWhite = MachineSettings(grinder = 4, strength = 7, volumeML = 35, milkSeconds = 18)

    val bean = BeanSnapshot(
        id = "sample-bean",
        name = "Sample Blend",
        roaster = "Test Roastery",
        roastLevel = RoastLevel.MEDIUM,
        milkFriendly = true,
        flavourNotes = listOf("chocolate", "caramel"),
        recommendedSettings = baselineFlatWhite,
    )
}
